scripts/tool/convert_image_to_data_url.py

```diff
@@ -5,6 +5,9 @@
 from PIL import Image
 
 
+SUPPORT_IMAGE_TYPES = ["png", "jpg", "jpeg", "gif", "bmp"]
```

Comment:
Merge `convert_image_to_data_url` functionality to generate package tool meta.

Reply:
Changed, thanks.